docs/user/commandline_tools.rst

```diff
@@ -424,6 +426,114 @@ Explain how a given GeoMesa query will be executed::
+      Stats analyzed:
+        Total features: 8852601
+        Bounds for geom: [ -171.75, -45.5903996, 157.7302, 89.99997102 ]
+        Bounds for dtg: [ '2016-02-01T00:09:12.000Z' to '2016-03-01T00:21:02.000Z' ]
+        Bounds for user_id: [ '100000215' to '99999502' ]
+      Use 'stats-histogram' or 'stats-count' commands for more details
+
+stats-bounds
+~~~~~~~~~~~~
+
+Displays the bounds of your data for different attributes. You can use pre-calculated stats for a quick
+estimation, or get the definitive result by querying the data set using the '--exact' flag.
+
+Example usage::
+
+    $ geomesa stats-bounds -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter
+      user_id [ 100000215 to 99999502 ] cardinality: 861283
+      user_name [ unavailable ]
+      text [ unavailable ]
+      dtg [ 2016-02-01T00:09:12.000Z to 2016-03-01T00:21:02.000Z ] cardinality: 2161132
+      geom [ -171.75, -45.5903996, 157.7302, 89.99997102 ] cardinality: 2119237
+
+    $ geomesa stats-bounds -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter --exact \
+        -q 'BBOX(geom,-70,45,-60,55) AND dtg DURING 2016-02-02T00:00:00.000Z/2016-02-03T00:00:00.000Z'
+      Running stat query...
+        user_id [ 1011811424 to 99124417 ] cardinality: 115
+        user_name [ bar_user to foo_user ] cardinality: 113
+        text [ bar to foo ] cardinality: 180
+        dtg [ 2016-02-02T00:01:07.000Z to 2016-02-02T23:59:41.000Z ] cardinality: 178
+        geom [ -69.87212338, 45.01259299, -60.08925, 53.8868369 ] cardinality: 155
+
+stats-count
+~~~~~~~~~~~
+
+Counts the features in your data set. You can count total features, or features that match a CQL filter.
+You can use pre-calculated stats for a quick estimation, or get the definitive result by querying the
+data set using the '--exact' flag.
+
+Example usage::
+
+    $ geomesa stats-count -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter
+      Estimated count: 8852601
+
+    $ geomesa stats-count -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter \
+        -q 'BBOX(geom,-70,45,-60,55) AND dtg DURING 2016-02-02T00:00:00.000Z/2016-02-03T00:00:00.000Z'
+      Estimated count: 2681
+
+    $ geomesa stats-count -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter --exact \
+        -q 'BBOX(geom,-70,45,-60,55) AND dtg DURING 2016-02-02T00:00:00.000Z/2016-02-03T00:00:00.000Z'
+      Running stat query...
+      Count: 182
+
+
+stats-histogram
+~~~~~~~~~~~~~~~
+
+Counts the features in your data set, grouped into sorted bins. You may specify the number of bins to group
+attribute into. You can count total features, or features that match a CQL filter. You can use
+pre-calculated stats for a quick estimation, or get the definitive result by querying the
+data set using the '--exact' flag.
```

Comment:
I find the string histogram a bit strange to look at in that it includes numbers.

e.g.
Binned histogram for 'actor2Name':
  [ 0 to 2x ] 4188
  [ 2x to 5u ] 3574
  [ 5u to 8r ] 3664
  [ 8r to bo ] 4354
  [ bo to el ] 5457
  [ el to hi ] 6158
  [ hi to kf ] 6928
  [ kf to nc ] 7402
  [ nc to q9 ] 8093
  [ q9 to t6 ] 11196
  [ t6 to w3 ] 11765
  [ w3 to z ] 2573114

In this case, actor2Name is the names of countries so there's no numbers involved. Also, where does the empty string go in this histogram? I assume after z? If so, that's inconsistent with the stats-analyze command which lists the empty string first: actor2Name [  to ZURICH ] cardinality: 6032